engine/table/src/test/java/io/deephaven/engine/table/impl/remote/TestConstructSnapshot.java

```diff
@@ -82,74 +82,5 @@ public UpdateGraph getUpdateGraph() {
         t2.join();
         assertEquals(1, changed.get());
     }
-
-    public void testUsePrevSnapshot() throws ExecutionException, InterruptedException {
```

Comment:
I wonder if we could re-do this in terms of barrage messages?

Reply:
This test seems like its testing the InitialSnapshot and InitialSnapshotTable behavior in terms of updates getting pushed downstream. So this seems very much specific to InitialSnapshot and I don't think we need to reimplement it.

Reply:
This should probably have been reimplemented in terms of BarrageMessage long ago. It's likely worth resurrecting, and I think the comparison tool you've created will allow doing so (maybe with a `usePrev` argument added).